analyses/cellranger-analysis/submit-multiple-jobs.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#BSUB -P run_cellranger
+#BSUB -J submitter
+#BSUB -q standard
+#BSUB -n 1
+#BSUB -R "rusage[mem=2500]"
+#BSUB -o submitter.out
+#BSUB -e submitter.err
+#BSUB -cwd "/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/sc-rna-seq-snap/analyses/cellranger-analysis"
+
+########################################################################
+# Set up running directory
+prefix="/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/sc-rna-seq-snap/analyses/cellranger-analysis"
+cd "$(dirname "${BASH_SOURCE[0]}")" 
+########################################################################
+# Create directories to save output files to
+mkdir -p ${prefix}/input
+mkdir -p ${prefix}/results
+mkdir -p ${prefix}/results/01_logs
+mkdir -p ${prefix}/results/02_cellranger_count
+mkdir -p ${prefix}/results/02_cellranger_count/DefaultParameters
+
+########################################################################
+# File in which we store the output text to verify the job execution order.
+# As the jobs will run on the cluster, we pass in a path to the directory from
+# which this launch script was run so everyone writes to the same file.
+output_file="output.txt"
+
+# Prints a simple LSF job file to standard output
+function print_job {
+    outfile=${1}
+    delay_seconds=${2}
+
+    # Preamble
+    echo "#!/bin/bash"
+
+    # Variables
+    echo "time=\`date\`"
+    echo "name=\${LSB_JOBNAME}"
+
+    # Describe job when it comes online, delay, then write completion text
+    echo "echo \"Online: name=\${name} time=\${time}\" >> ${outfile}"
+    echo "sleep ${delay_seconds}"
+    echo "echo \"Done: name=\${name}\" >> ${outfile}"
```

Comment:
I think adding the prefix path here (and in line 43) would fix the output.txt errors from j1.bsub, similar to how it fixed the j1 job command below

Reply:
I added the [refix, let's test it to see what happens. See e4a55f44e2904287a0865f0098ba8a56c15195c5.
 I also reported is an issue here #17.

Reply:
Adding the prefix appears to have fixed the issue, the output.txt file is now generated for the j1 job.

Reply:
There is also #16 issue that explains a problem with folder permissions. This was not affecting the run and results, but it was addressed as well here https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap/commit/e4a55f44e2904287a0865f0098ba8a56c15195c5.